lib/mayaUsd/fileio/jobs/jobArgs.h

```diff
@@ -270,7 +270,7 @@ struct UsdMayaJobExportArgs
     const TfToken::Set includeAPINames;
     const TfToken::Set jobContextNames;
     const TfToken::Set excludeExportTypes;
-    const std::string  defaultPrim;
+    std::string        defaultPrim;
```

Comment:
When staring to export, when no default prim were specified a default prim will be set, so the variable must no be const anymore.